src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentClient.kt

```diff
@@ -100,17 +100,20 @@ class CodyAgentClient(private val project: Project, private val webview: NativeW
 
   @JsonRequest("textDocument/show")
   fun textDocument_show(params: TextDocument_ShowParams): CompletableFuture<Boolean> {
-    return acceptOnEventThreadAndGet {
-      val selection = params.options?.selection
-      val preserveFocus = params.options?.preserveFocus
-      val vf = CodyEditorUtil.findFileOrScratch(project, params.uri)
-      if (vf != null) {
-        CodyEditorUtil.showDocument(project, vf, selection, preserveFocus)
-        true
-      } else {
-        false
-      }
-    }
+    val vf =
+        acceptOnEventThreadAndGet { CodyEditorUtil.findFileOrScratch(project, params.uri) }.get()
+
+    val result =
+        if (vf != null) {
+          val selection = params.options?.selection
+          val preserveFocus = params.options?.preserveFocus
+          CodyEditorUtil.showDocument(project, vf, selection, preserveFocus)
+          true
```

Comment:
We can delate that `true` as it is incorrect, `showDocument` already returns `true` if it succeeded.

Reply:
fixed